src/Components/Artwork/GridItem.tsx

```diff
@@ -28,11 +28,52 @@ interface Props extends RelayProps, React.HTMLProps<ArtworkGridItemContainer> {
   }
 }
 
-class ArtworkGridItemContainer extends React.Component<Props, null> {
+interface State {
+  width: number
+  height: number
+}
+
+const GEMINI_HOST = "d7hftxdivxxvm.cloudfront.net"
+const IMAGE_QUALITY = 80
+
+class ArtworkGridItemContainer extends React.Component<Props, State> {
   static defaultProps = {
     useRelay: true,
   }
 
+  image: HTMLImageElement = null
```

Comment:
I'm a bit unsure exactly what this is, is this just a private instance variable?

Reply:
Yup just an instance variable, though not private; will prefix.